llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -610,22 +610,134 @@ static MCCFIInstruction createDefCFAOffset(const TargetRegisterInfo &TRI,
                                         Comment.str());
 }
 
+// Allocate stack space and probe it if necessary.
 void RISCVFrameLowering::allocateStack(MachineBasicBlock &MBB,
                                        MachineBasicBlock::iterator MBBI,
-                                       StackOffset Offset, bool EmitCFI,
-                                       unsigned CFIIndex) const {
```

Comment:
I see the CFIIndex argument was removed. Can that be done as a separate patch?

Reply:
Ok, I created a opened a new one: https://github.com/llvm/llvm-project/pull/117871